src/amo/components/AddonsCard/styles.scss

```diff
@@ -20,6 +20,18 @@
       border: 0;
       position: absolute;
       top: 0;
+
+      @include respond-to(large) {
+          [dir='ltr'] & {
+              padding-left:0;
+              left: 50%;
```

Comment:
Did you try just using `width: 50%`? That seems a bit simpler than using `left` and `right` based on `dir`.

Reply:
You're right it's much easier. I should have used that :smile: